### PR TITLE
Edit templates to show time zone in event details section.

### PR DESCRIPTION
**Pull request by Camwyn**
📷 (unchecked): http://p.tri.be/suAGOc
📸 (checked): http://p.tri.be/mgeEcn

[TEC-2979]

[TEC-2979]: https://moderntribe.atlassian.net/browse/TEC-2979

### CODE REVIEW

File: src/views/blocks/parts/details.php [hunk 1]
```diff
@@ -122,6 +123,9 @@
 			<dd>
 				<div class="tribe-events-abbr tribe-events-start-time published dtstart" title="<?php esc_attr_e( $end_ts ) ?>">
 					<?php echo $time_formatted; ?>
+					<?php if ( $show_time_zone ) : ?>
+						<span class="tribe-events-abbr tribe-events-time-zone published "><?php echo esc_html( $time_zone_label ); ?></span>
+					<?php endif; ?>
```
Comment (paulmskim) [inline src/views/blocks/parts/details.php:128]: Should this be under multiday events too?

File: src/views/modules/meta/details.php [hunk 2]
```diff
@@ -112,6 +113,9 @@
 			<dd>
 				<div class="tribe-events-abbr tribe-events-start-time published dtstart" title="<?php esc_attr_e( $end_ts ) ?>">
 					<?php echo $time_formatted; ?>
+					<?php if ( $show_time_zone ) : ?>
+						<span class="tribe-events-abbr tribe-events-time-zone published "><?php echo esc_html( $time_zone_label ); ?></span>
+					<?php endif; ?>
```
Comment (paulmskim) [inline src/views/modules/meta/details.php:118]: Should this be under multiday events too?

File: src/views/blocks/parts/details.php [hunk 3]
```diff
@@ -58,9 +59,9 @@
 <div class="tribe-events-meta-group tribe-events-meta-group-details">
 	<h3 class="tribe-events-single-section-title">
 		<?php if ( empty( $attributes['detailsTitle'] ) ) : ?>
-			<?php esc_html_e( 'Details', 'the-events-calendar' ) ?>
+			<?php esc_html_e( 'Details', 'the-events-calendar' ); ?>
 		<?php else : ?>
-			<?php echo is_array( $attributes['detailsTitle'] ) ? reset( $attributes['detailsTitle'] ) : esc_html( $attributes['detailsTitle'] ) ?>
+			<?php echo is_array( $attributes['detailsTitle'] ) ? reset( $attributes['detailsTitle'] ) : esc_html( $attributes['detailsTitle'] ); ?>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:64]: * :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'reset'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 51e2c2ca1a15c99051be7cff0b74749d</sub>

File: src/views/blocks/parts/details.php [hunk 4]
```diff
@@ -73,55 +74,65 @@
 		if ( tribe_event_is_all_day() && tribe_event_is_multiday() ) :
 			?>
 
-			<dt> <?php esc_html_e( 'Start:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'Start:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-start-datetime published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-start-datetime published dtstart" title="<?php esc_attr_e( $start_ts ); ?>"> <?php esc_html_e( $start_date ); ?> </abbr>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:79]: * :no_entry_sign: The $text arg must be a single string literal, not "$start_ts".
* :no_entry_sign: The $text arg must be a single string literal, not "$start_date".

<sub>hash: 5c1e0d5b3d21956f7131a0c7ad183a84</sub>

File: src/views/blocks/parts/details.php [hunk 5]
```diff
@@ -73,55 +74,65 @@
...
+
 			</dd>
 
-			<dt> <?php esc_html_e( 'End:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'End:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr dtend" title="<?php esc_attr_e( $end_ts ) ?>"> <?php esc_html_e( $end_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr dtend" title="<?php esc_attr_e( $end_ts ); ?>"> <?php esc_html_e( $end_date ); ?> </abbr>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:85]: * :no_entry_sign: The $text arg must be a single string literal, not "$end_ts".
* :no_entry_sign: The $text arg must be a single string literal, not "$end_date".

<sub>hash: 20849db963971bd9ad2e9ebdc3fbde97</sub>

File: src/views/blocks/parts/details.php [hunk 6]
```diff
@@ -73,55 +74,65 @@
...
 		<?php
 		// All day (single day) events
 		elseif ( tribe_event_is_all_day() ):
 			?>
 
-			<dt> <?php esc_html_e( 'Date:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'Date:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-start-datetime published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-start-datetime published dtstart" title="<?php esc_attr_e( $start_ts ); ?>"> <?php esc_html_e( $start_date ); ?> </abbr>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:95]: * :no_entry_sign: The $text arg must be a single string literal, not "$start_ts".
* :no_entry_sign: The $text arg must be a single string literal, not "$start_date".

<sub>hash: 9d5fd0d7232b98c14e868aca3385156f</sub>

File: src/views/blocks/parts/details.php [hunk 7]
```diff
@@ -73,55 +74,65 @@
...
 		<?php
 		// Multiday events
 		elseif ( tribe_event_is_multiday() ) :
 			?>
 
-			<dt> <?php esc_html_e( 'Start:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'Start:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr updated published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_datetime ) ?> </abbr>
+				<abbr class="tribe-events-abbr updated published dtstart" title="<?php esc_attr_e( $start_ts ); ?>"> <?php esc_html_e( $start_datetime ); ?> </abbr>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:105]: * :no_entry_sign: The $text arg must be a single string literal, not "$start_ts".
* :no_entry_sign: The $text arg must be a single string literal, not "$start_datetime".

<sub>hash: beec266548452e71b74b5e7d9d100b11</sub>

File: src/views/blocks/parts/details.php [hunk 8]
```diff
@@ -73,55 +74,65 @@
...
+				<?php if ( $show_time_zone ) : ?>
+					<span class="tribe-events-abbr tribe-events-time-zone published"><?php echo esc_html( $time_zone_label ); ?></span>
+				<?php endif; ?>
 			</dd>
 
-			<dt> <?php esc_html_e( 'End:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'End:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr dtend" title="<?php esc_attr_e( $end_ts ) ?>"> <?php esc_html_e( $end_datetime ) ?> </abbr>
+				<abbr class="tribe-events-abbr dtend" title="<?php esc_attr_e( $end_ts ); ?>"> <?php esc_html_e( $end_datetime ); ?> </abbr>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:113]: * :no_entry_sign: The $text arg must be a single string literal, not "$end_ts".
* :no_entry_sign: The $text arg must be a single string literal, not "$end_datetime".

<sub>hash: 6eef9345b2d50e5f10119dad290fb32e</sub>

File: src/views/blocks/parts/details.php [hunk 9]
```diff
@@ -73,55 +74,65 @@
...
 		<?php
 		// Single day events
 		else :
 			?>
 
-			<dt> <?php esc_html_e( 'Date:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'Date:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-start-date published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-start-date published dtstart" title="<?php esc_attr_e( $start_ts ); ?>"> <?php esc_html_e( $start_date ); ?> </abbr>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:126]: * :no_entry_sign: The $text arg must be a single string literal, not "$start_ts".
* :no_entry_sign: The $text arg must be a single string literal, not "$start_date".

<sub>hash: 740d54064e96c0988ddd889b15bb9a82</sub>

File: src/views/blocks/parts/details.php [hunk 10]
```diff
@@ -73,55 +74,65 @@
...
 			</dd>
 
 			<dt> <?php echo esc_html( $time_title ); ?> </dt>
 			<dd>
-				<div class="tribe-events-abbr tribe-events-start-time published dtstart" title="<?php esc_attr_e( $end_ts ) ?>">
+				<div class="tribe-events-abbr tribe-events-start-time published dtstart" title="<?php esc_attr_e( $end_ts ); ?>">
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:131]: :no_entry_sign: The $text arg must be a single string literal, not "$end_ts".

<sub>hash: e67b7b14704d6578b6fe0c11df4aa901</sub>

File: src/views/blocks/parts/details.php [hunk 11]
```diff
@@ -131,7 +142,7 @@
 		// Event Cost
 		if ( ! empty( $cost ) ) : ?>
 
-            <dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ) ?> </dt>
+            <dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ); ?> </dt>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:145]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 3d4196da3416e259e93f24c1c24c3f87</sub>

File: src/views/blocks/parts/details.php [hunk 12]
```diff
@@ -150,16 +161,16 @@
 		);
 		?>
 
-		<?php echo tribe_meta_event_tags( sprintf( esc_html__( '%s Tags:', 'the-events-calendar' ), tribe_get_event_label_singular() ), ', ', false ) ?>
+		<?php echo tribe_meta_event_tags( sprintf( esc_html__( '%s Tags:', 'the-events-calendar' ), tribe_get_event_label_singular() ), ', ', false ); ?>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:164]: * :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'tribe_meta_event_tags'.
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 63bd19beffb426edac58cc6fafa785eb</sub>

File: src/views/blocks/parts/details.php [hunk 13]
```diff
@@ -131,8 +142,8 @@
 		// Event Cost
 		if ( ! empty( $cost ) ) : ?>
 
-            <dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ) ?> </dt>
-            <dd class="tribe-events-event-cost"> <?php esc_html_e( $cost ); ?> </dd>
+            <dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ); ?> </dt>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:145]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 3f1bef08e5c14f3b87963e750bc36aa6</sub>

File: src/views/blocks/parts/details.php [hunk 14]
```diff
@@ -131,8 +142,8 @@
...
+            <dd class="tribe-events-event-cost"> <?php echo esc_html( $cost ); ?> </dd>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:146]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: efb3d56bb3585c442ca1eaf70e0ed194</sub>

File: src/views/blocks/parts/details.php (hunk 12 above)
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:164]: * :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'tribe_meta_event_tags'.
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: db3da7aa62a72b1c664c9a0763e5947f</sub>

File: src/views/modules/meta/details.php [hunk 15]
```diff
@@ -65,12 +66,12 @@
 
 			<dt class="tribe-events-start-date-label"> <?php esc_html_e( 'Start:', 'the-events-calendar' ) ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-start-date published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-start-date published dtstart" title="<?php echo esc_attr( $start_ts ) ?>"> <?php echo esc_html( $start_date ) ?> </abbr>
```
Comment (tr1b0t) [inline src/views/modules/meta/details.php:69]: * :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: c35fe2c8f59d62627dc5cae68699b543</sub>

File: src/views/modules/meta/details.php [hunk 16]
```diff
@@ -65,12 +66,12 @@
...
 			</dd>
 
 			<dt class="tribe-events-end-date-label"> <?php esc_html_e( 'End:', 'the-events-calendar' ) ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-end-date dtend" title="<?php esc_attr_e( $end_ts ) ?>"> <?php esc_html_e( $end_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-end-date dtend" title="<?php echo esc_attr( $end_ts ) ?>"> <?php echo esc_html( $end_date ) ?> </abbr>
```
Comment (tr1b0t) [inline src/views/modules/meta/details.php:74]: * :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: 87be2ea5aff572866ab3e67e6f5e4506</sub>

File: src/views/modules/meta/details.php [hunk 17]
```diff
@@ -80,7 +81,7 @@
 
 			<dt class="tribe-events-start-date-label"> <?php esc_html_e( 'Date:', 'the-events-calendar' ) ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-start-date published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-start-date published dtstart" title="<?php echo esc_attr( $start_ts ) ?>"> <?php echo esc_html( $start_date ) ?> </abbr>
```
Comment (tr1b0t) [inline src/views/modules/meta/details.php:84]: * :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: aa7cf35b2d52cac7f9ed2138cb3792df</sub>

File: src/views/modules/meta/details.php [hunk 18]
```diff
@@ -90,12 +91,18 @@
 
 			<dt class="tribe-events-start-datetime-label"> <?php esc_html_e( 'Start:', 'the-events-calendar' ) ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-start-datetime updated published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_datetime ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-start-datetime updated published dtstart" title="<?php echo esc_attr( $start_ts ) ?>"> <?php echo esc_html( $start_datetime ) ?> </abbr>
```
Comment (tr1b0t) [inline src/views/modules/meta/details.php:94]: * :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: a701c6e683469bfd752e10a6d88824e8</sub>

File: src/views/modules/meta/details.php [hunk 19]
```diff
@@ -90,12 +91,18 @@
...
+				<?php if ( $show_time_zone ) : ?>
+					<span class="tribe-events-abbr tribe-events-time-zone published "><?php echo esc_html( $time_zone_label ); ?></span>
+				<?php endif; ?>
 			</dd>
 
 			<dt class="tribe-events-end-datetime-label"> <?php esc_html_e( 'End:', 'the-events-calendar' ) ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-end-datetime dtend" title="<?php esc_attr_e( $end_ts ) ?>"> <?php esc_html_e( $end_datetime ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-end-datetime dtend" title="<?php echo esc_attr( $end_ts ) ?>"> <?php echo esc_html( $end_datetime ) ?> </abbr>
```
Comment (tr1b0t) [inline src/views/modules/meta/details.php:102]: * :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: 92cbf0b972f250f3e3f8d54c7334b52c</sub>

File: src/views/modules/meta/details.php [hunk 20]
```diff
@@ -105,13 +112,16 @@
 
 			<dt class="tribe-events-start-date-label"> <?php esc_html_e( 'Date:', 'the-events-calendar' ) ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-start-date published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-start-date published dtstart" title="<?php echo esc_attr( $start_ts ) ?>"> <?php echo esc_html( $start_date ) ?> </abbr>
```
Comment (tr1b0t) [inline src/views/modules/meta/details.php:115]: * :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: 5a8826ba68ba82721cfa020c64b215e6</sub>

File: src/views/modules/meta/details.php [hunk 21]
```diff
@@ -105,13 +112,16 @@
...
 			</dd>
 
 			<dt class="tribe-events-start-time-label"> <?php echo esc_html( $time_title ); ?> </dt>
 			<dd>
-				<div class="tribe-events-abbr tribe-events-start-time published dtstart" title="<?php esc_attr_e( $end_ts ) ?>">
+				<div class="tribe-events-abbr tribe-events-start-time published dtstart" title="<?php echo esc_attr( $end_ts ) ?>">
```
Comment (tr1b0t) [inline src/views/modules/meta/details.php:120]: :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: b6efddd880fc3ef3c7b553f2b908b04b</sub>

File: src/views/blocks/parts/details.php [hunk 22]
```diff
@@ -130,12 +141,11 @@
 		<?php
 		// Event Cost
 		if ( ! empty( $cost ) ) : ?>
-
-            <dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ) ?> </dt>
-            <dd class="tribe-events-event-cost"> <?php esc_html_e( $cost ); ?> </dd>
+			<dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ); ?> </dt>
+			<dd class="tribe-events-event-cost"> <?php echo esc_html( $cost ); ?> </dd>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:145]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 6696a3c83d46016669e910bc03396894</sub>

File: src/views/blocks/parts/details.php [hunk 23]
```diff
@@ -130,12 +141,11 @@
...
 		<?php endif ?>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:146]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: ec6508071c446393c5f8c4e6ebeee466</sub>

File: src/views/blocks/parts/details.php [hunk 24]
```diff
@@ -150,16 +160,16 @@
 		);
 		?>
 
-		<?php echo tribe_meta_event_tags( sprintf( esc_html__( '%s Tags:', 'the-events-calendar' ), tribe_get_event_label_singular() ), ', ', false ) ?>
+		<?php tribe_meta_event_tags( sprintf( esc_html__( '%s Tags:', 'the-events-calendar' ), tribe_get_event_label_singular() ), ', ', true ); ?>
```
Comment (tr1b0t) [inline src/views/blocks/parts/details.php:164]: :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 3f5e6f9a4db63e86cb7a9b8967fc196a</sub>

File: src/views/modules/meta/details.php [hunk 25]
```diff
@@ -140,16 +150,16 @@
 		);
 		?>
 
-		<?php echo tribe_meta_event_tags( sprintf( esc_html__( '%s Tags:', 'the-events-calendar' ), tribe_get_event_label_singular() ), ', ', false ) ?>
+		<?php tribe_meta_event_tags( sprintf( esc_html__( '%s Tags:', 'the-events-calendar' ), tribe_get_event_label_singular() ), ', ', true ); ?>
```
Comment (tr1b0t) [inline src/views/modules/meta/details.php:153]: :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: b5a5ead796ece02a715277a424e72faa</sub>

File: src/views/blocks/parts/details.php (hunk 22 above)
Comment (Camwyn) [reply src/views/blocks/parts/details.php:145]: I think the bot is tripping on the space _inside_ the <dd>?

**Pull request merged**